Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM python:3.11
+FROM python:3.10-slim
```

Comment:
Python bump is and using a slim image are separate design decisions. Use 2 separate PRs for this single line as it's easier to approve and merge them.